### PR TITLE
terraform_0_12: 0.12.5 -> 0.12.6

### PR DESCRIPTION
**Pull request by rawkode**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @zimbatm @peterhoeg @kalbasit

### CODE REVIEW

File: pkgs/applications/networking/cluster/terraform/default.nix [hunk 1]
```diff
@@ -89,16 +89,16 @@ let
 in rec {
   terraform_0_11 = pluggable (generic {
     version = "0.11.14";
-    sha256 = "1bzz5wy13gh8j47mxxp6ij6yh20xmxd9n5lidaln3mf1bil19dmc";
+    sha256 = "";
```
Comment (kalbasit) [inline pkgs/applications/networking/cluster/terraform/default.nix]: please revert this change.

**Review by kalbasit — CHANGES_REQUESTED** [review]

File: pkgs/applications/networking/cluster/terraform/default.nix (hunk 1 above)
Comment (rawkode) [reply pkgs/applications/networking/cluster/terraform/default.nix]: Oops, fixed @kalbasit

**Comment by kalbasit** [comment]
@GrahamcOfBorg buid terraform_0_12 terraform_0_12.full

**Comment by kalbasit** [comment]
@GrahamcOfBorg build terraform_0_12 terraform_0_12.full

**Review by kalbasit — APPROVED** [review]
